likecoin/js/admin-settings/src/pages/LikecoinButtonPage.js

```diff
@@ -68,6 +70,7 @@ function LikecoinButtonPage() {
     () => debounce(async (likerId) => {
       setSavedSuccessful(false);
       setIsLoading(true);
+      if (!likerId) return; // user did not input any value. Want to forceDisconnect.
```

Comment:
Why not just set `forceDisconnect` to true/false here?

Reply:
It was to enable the hacky way for empty likerId value. Will change this.